src/config.py

```diff
@@ -8,13 +8,16 @@
 
 
 load_dotenv()
-ETHEREUM_NODE_URL = os.getenv("ETHEREUM_NODE_URL")
-GNOSIS_NODE_URL = os.getenv("GNOSIS_NODE_URL")
-
-CHAIN_RPC_ENDPOINTS = {"Ethereum": ETHEREUM_NODE_URL, "Gnosis": GNOSIS_NODE_URL}
+NODE_URL = os.getenv("NODE_URL")
 
 logger = get_logger("raw_token_imbalances")
 
+# Utilized by imbalances_script for computing for single tx hash
+CHAIN_RPC_ENDPOINTS = {
```

Comment:
If its okay, I've left this since we use it when solely running the imbalances_script on a single tx hash to find which chain it belongs to. Otherwise, I can remove this and also main() in imbalances_script, and just use a new file locally to test on single hashes.